lib/index.js

```diff
@@ -41,8 +46,9 @@ function register (server, options, next) {
     server.method('pg.people.getAllPeople', getAllPeople);
     server.method('pg.people.getBy', peopleGetBy);
     server.method('pg.organisations.addOrgName', addOrgName);
-    server.method('pg.organisations.getActiveOrgs', getActiveOrgs);
-    server.method('pg.organisations.getById', getActiveOrgs);
+    server.method('pg.organisations.orgsGetByTag', orgsGetByTag);
+    // COMMENTED BELOW LINE OUT AS DOES NOT YET EXIST
```

Comment:
it's better to delete the lines instead of commenting them to keep the code clean.